composer/loggers/mosaicml_logger.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Log to the MosaicML platform."""
+
+from __future__ import annotations
+
+import collections.abc
+import logging
+import operator
+import os
+import time
+from functools import reduce
+from typing import TYPE_CHECKING, Any, Dict, Optional
+
+import torch
+
+from composer.loggers.logger import Logger
+from composer.loggers.logger_destination import LoggerDestination
+from composer.utils import MissingConditionalImportError, dist
+
+if TYPE_CHECKING:
+    from composer.core import State
+
+log = logging.getLogger(__name__)
+
+__all__ = ['MosaicMLLogger']
+
+RUN_NAME_ENV_VAR = 'RUN_NAME'
+
+
+class MosaicMLLogger(LoggerDestination):
```

Comment:
should we have a `filter_keys` or something (i.e. if you want to avoid sending optimizer monitor logs to mcli or something)

Reply:
Good call... yea let me add something + unit test it